src/spikeinterface/core/template.py

```diff
@@ -165,6 +165,67 @@ def from_dict(cls, data):
             probe=data["probe"] if data["probe"] is None else Probe.from_dict(data["probe"]),
         )
 
+    def save_to_zarr(self, folder_path):
+        import zarr
+
+        zarr_group = zarr.open_group(folder_path, mode="w")
+
+        # Saves one chunk per unit
+        arrays_chunk = (1, None, None)
+        zarr_group.create_dataset("templates_array", data=self.templates_array, chunks=arrays_chunk)
+        zarr_group.create_dataset("channel_ids", data=self.channel_ids)
+        zarr_group.create_dataset("unit_ids", data=self.unit_ids)
+
+        zarr_group.attrs["sampling_frequency"] = self.sampling_frequency
+        zarr_group.attrs["nbefore"] = self.nbefore
+
+        if self.sparsity_mask is not None:
+            zarr_group.create_dataset("sparsity_mask", data=self.sparsity_mask)
+
+        if self.probe is not None:
+            probe_dict = self.probe.to_dict(array_as_list=True)
+            zarr_group.attrs["probe"] = probe_dict
+
+    @classmethod
+    def load_from_open_zarr_storage(cls, base_root):
```

Comment:
```suggestion
    def load_from_zarr_group(cls, base_root):
```
A zarr open storage is a zarr group, no?

Reply:
It doesn't need to be root. For example, the new sorting result could save this in the `results/templates` group!

Reply:
OK, let's go for the suggested naming and say that the structure of the templates as defined by the writing function should anchored in that group.

Reply:
Done.